tests/client/test_cursor.py

```diff
@@ -205,7 +205,16 @@ def test_execute_with_converter(self):
                 [
                     "foo",
                     IPv4Address("10.10.10.1"),
-                    datetime.datetime(2022, 7, 18, 18, 10, 36, 758000),
+                    datetime.datetime(
+                        2022,
+                        7,
+                        18,
+                        18,
+                        10,
+                        36,
+                        758000,
+                        tzinfo=datetime.timezone.utc,
```

Comment:
Do we need to test other time zones too?

Reply:
The test suite bears a few other test cases that do something with non-UTC time zones. Do you like them?

https://github.com/crate/crate-python/blob/f0ef825c42865a29084f11de1107814eda7a5acd/tests/client/test_cursor.py#L46-L148

Reply:
Thanks! All I need 😄